### ISSUE TITLE
Separate syncing of gamemode and flight status

### ISSUE THREAD

**Issue by Dakotaa** [WiIIiam278/HuskSync#191]
SaberFactions can give players Factions fly, which enables flight for them. It seems that in the latest HuskSync, if they have factions fly enabled in the Factions world and switch to another world, their fly syncs and they can fly in their destination world. Seems to be linked to gamemode syncing as it's stopped now that I've turned that off, but this is not an ideal solution. 

Perhaps a separate sync option for flight status?

**Comment by WiIIiam278** [comment]
yeah basically we'd have to re-separate 'em out entirely. probably an oversight on my part when I went and compartmentalized some of the modes to simplify logic
